app/controllers/first_draft_collections_controller.rb

```diff
@@ -0,0 +1,67 @@
+# typed: false
+# frozen_string_literal: true
+
+# Handles updates for first drafts of collections.
+# When updating a first draft of a collection, you need to show a form that includes both
+# the settings and the details all in one page.  For later versions, the form is split in two.
+class FirstDraftCollectionsController < ObjectsController
+  before_action :authenticate_user!
+  before_action :ensure_sdr_updatable
+
+  def edit
```

Comment:
Should we move `new` and `create` here as well since they're also working on the `CreateCollectionForm`?

Reply:
Yeah, I thought about that too.  Let me give it a try and see how many changes it actually invokes.